index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>freeCodeCamp Survey Form</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <main>
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+        <form action="https://freecodecamp.com/submit-survey-form" id="survey-form">
+            <div>
+            <section>
+                <fieldset class="top">
+                    <label id="name-label">Name: <input type="text" ; placeholder="Enter your name" id="name"
+                            required></label>
+                    <label id="email-label">Email: <input type="email" placeholder="Enter your Email" id="email"
+                            required></input></label>
+                    <label id="number-label">Age: <em class="embed">(optional)</em> <input type="number" placeholder="Age" id="number" min="16"
+                            max="80"></label>
+                </fieldset>
+            </section>
+            <section>
+                <fieldset>
+                    <legend>Which option best describes your current role?</legend>
+                    <select class="dropdown" id="dropdown">
```

Comment:
- I think it would be better to add a `title` element to the `select` element for accessibility purposes incase screen readers are used.